### PR TITLE
add pkgdown site

### LINKED ISSUE #8: Add a pkgdown site
You should add a pkdown site and host it on github-pages

### PR DESCRIPTION
**Pull request by BruceZhaoR**
fix #8

### CODE REVIEW

**Comment by BruceZhaoR** [comment]
@jameslamb Maybe you need to modify your git name and email in `pkgdown/deploy.sh` and add GITHUB_PAT to Appveyor `project` -> `settings` -> `Environment variables`

**Comment by jameslamb** [comment]
oh I forgot the GITHUB_PAT! Yep will do that later today

File: appveyor.yml [hunk 1]
```diff
@@ -8,19 +8,37 @@ init:
...
 build_script:
+  - pandoc -v
   - travis-tool.sh install_deps
+  - Rscript.exe --verbose ./pkgdown/build-site.R
 
 test_script:
   - travis-tool.sh run_tests
 
+on_finish:
+  - bash.exe ./pkgdown/deploy.sh
```
Comment (jameslamb) [inline appveyor.yml:40]: this is amazing

File: appveyor.yml [hunk 2]
```diff
@@ -8,19 +8,37 @@ init:
...
 install:
-  ps: Bootstrap
+  - ps: Bootstrap
+  - cinst pandoc
+  - ps: $env:Path += ";C:\Program Files\Pandoc\"
 
 cache:
   - C:\RLibrary
+  - C:\ProgramData\chocolatey\bin -> appveyor.yml
+  - C:\ProgramData\chocolatey\lib -> appveyor.yml
```
Comment (jameslamb) [inline appveyor.yml:18]: what is `chocolatey` and what does the `-> appveyor.yml` do? (I'm not very familiar with powershell)

**Comment by jameslamb** [comment]
I put my GITHUB_PAT up on Appveyor! Going to do a rebuild and see if it deploys

**Review by jameslamb — APPROVED** [review]

**Comment by jameslamb** [comment]
WOW that totally worked!!!!!

https://jameslamb.github.io/rsixygen/

You're a magician @BruceZhaoR 😀

**Pull request merged**